src/core/data/ions/particle_initializers/maxwellian_particle_initializer.hpp

```diff
@@ -44,13 +44,13 @@ class MaxwellianParticleInitializer : public ParticleInitializer<ParticleArray,
         std::uint32_t const& nbrParticlesPerCell, std::optional<std::size_t> seed = {},
         Basis const basis                                = Basis::Cartesian,
         std::array<InputFunction, 3> const magneticField = {nullptr, nullptr, nullptr},
-        InputFunction const ppc_by_icell                 = nullptr /*used if set */)
+        double densityCutOff                             = 1e-5)
```

Comment:
I know it's WIP, but just not to forget: add this to python API, maybe in the MaxwellianFluidModel population dict